components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx

```diff
@@ -200,6 +210,7 @@ const renderLicenseContent = (
             <hr/>
             {renderAddNewLicenseButton(fileInputRef, handleChange)}
             {renderRemoveButton(handleRemove, isDisabled, removing)}
+            <button onClick={openPreTrialPricingModal}>{'Open Pre trial pricing'}</button>
```

Comment:
This is a temporary addition to facilitate opening the pricing modal

Reply:
Even though this PR may cover only the Pre-Trial pricing behavior, we should only have one pricing Modal compnent. Suggesting to remove the pretrial references and just name it `PricingModal`